releases/net472/KnownIssues/611802 - SQLConnection.ConnectionString throws NullReferenceException .md

```diff
@@ -20,4 +20,6 @@ No known workarounds with .NET Framework 4.7.2.
 
 ## Resolution
 
-The .NET team is working on a fix that will be available in a future update.
+This issue has been fixed in the latest update to .NET Framework 4.7.2 [here](http://go.microsoft.com/fwlink/?LinkId=2005309), for all platforms except Windows 10 April 2018 Update. 
```

Comment:
I think it might be better to have present the link and link test as: \[the latest update to .NET Framework 4.7.2](http://go.microsoft.com/fwlink/?LinkId=2005309).
Also, no comma before for

Reply:
Will fix